### DISCUSSION TITLE
New Personal Avatar Functions

### DISCUSSION THREAD

**Discussion by SilverMoonDev** [vrcx-team/VRCX#169] [Ideas]
Can you add a "Delete Avatar" and "Change Image" button?
I will really appreciate it

**Comment by SilverMoonDev** [comment]
API.deleteAvatar = function (params) {
        return this.call(`avatars/${params.avatarId}`, {
            method: 'DELETE',
            params
        }).then((json) => {
            var args = {
                json,
                params
            };
            this.$emit('AVATAR:DELETE', args);
            return args;
        });
    };

**Reply by Natsumi-sama** [reply]
I was getting there, but okay haha

**Comment by SilverMoonDev** [comment]
case 'Delete Avatar':
                            API.deleteAvatar({
                                avatarId: D.id
                            }).then((args) => {
                                this.$message({
                                    message: 'Avatar deleted',
                                    type: 'success'
                                });
                                return args;
                            });
                            break;

**Comment by Natsumi-sama** [comment]
https://github.com/pypy-vrc/VRCX/commit/267ce200f6515c7e0cd306d163f7e64b8dded972 https://github.com/pypy-vrc/VRCX/commit/26fa43a311219ffcba1b44e61dae72ec61885fcc
